pushjack/apns.py

```diff
@@ -388,9 +402,25 @@ def check_error(self, timeout=10):
 
     def send(self, frames):
         """Send stream of frames to APNS server."""
+        current_identifier = frames.next_identifier
```

Comment:
There are two possible issues that can be caused by this that come to my mind. The first is that if someone would access the next_identifier ivar inside the for loop, it would contain a wrong value, which can help us, but the value won't actually be a next identifier. The second is that we would be relying on the code to always go thru all the list, because if it would break somewhere, the next_identifier ivar would contain a wrong value.

Reply:
Yes, you are right.